Software Documentation/Naming and Coding Style Conventions.md

```diff
@@ -0,0 +1,48 @@
+### Variables
+- Constants
+    - Preceded by `k`
+    - First word should be the name of the subsystem the constant is for
+    - Ex: `kDriveLeftRearTalonId`, `kLimelightFov`, `kControlLoopPeriod`
+- Fields
+    - Preceded by `m`
+    - Fields are any variables found in the **class body**
+    - Ex: `mWristAngle`, `mGyroAngle`
+- Parameters
+    - Preceded by `p`
+    - Ex:
+    ```
+        public DriveStraight(Drive pDrive, double pDistanceToDrive) {
+            mDrive = pDrive;
+            mDistanceToDrive = pDistanceToDrive;
+        }
+    ```
+
+### Constructors
+- If your class needs to use another classes' object, it should be passed into the constructor
```

Comment:
Instead of saying that you'll pass the specific classes, refer to them as dependencies. Also if the class implements an interface, the interface should be passed in to decouple from specific implementations.